### PR TITLE
feat: for GDPR compliance we need user to explicitly agree to privacy policy on sign in

### PR DESCRIPTION
**Pull request by chinook25**

### CODE REVIEW

File: apps/central/src/components/admin/ManagePrivacyPolicy.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <h3>Privacy policy</h3>
+    <div>
+      <button
```
Comment (mswertz) [inline apps/central/src/components/admin/ManagePrivacyPolicy.vue:5]: I would expect that you use a checkbox or select input here? Curious why you make this choice

**Review by mswertz — COMMENTED** [review]
just a small inline comment. Happy to see this developing.

File: apps/central/src/components/admin/ManagePrivacyPolicy.vue (hunk 1 above)
Comment (chinook25) [reply apps/central/src/components/admin/ManagePrivacyPolicy.vue:5]: If it was a checkbox list or select we'd need to track whether the text field is dirty, so we can deselect that chosen option. We chose buttons because it was simpler to implement for now (and since there was still some back and forthing over the story). 
Of course this is still a draft PR so if we'd want it differently we can change it, but we need to know what exactly we want for it ;)

File: apps/central/src/components/admin/ManagePrivacyPolicy.vue [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      loading: true,
+    };
+  },
+  methods: {
+    updatePolicyLevel(newValue) {
+      this.policyLevel = newValue;
+      this.policyText = PREFABS[newValue];
+    },
+    async fetchPrivacyPolicy() {
+      const response = await request("graphql", `{_settings{key, value}}`);
```
Comment (connoratrug) [inline apps/central/src/components/admin/ManagePrivacyPolicy.vue:58]: maybe we should move this to a client function ( can be separate PR)

**Review by connoratrug — COMMENTED** [review]
Code looks good

- not sure about the text in the policy box ( data + medical ?? ), seems confusing to me , as this is the default ? 
- the message is only shown on signin , but what about users that do not sign in

**Review by mswertz — DISMISSED** [review]
looks almost good. Now it is time to force Ulrike to produce at least one of these texts. Otherwise we cannot put this into production.

**Review by MaxPostema — APPROVED** [review]
